text/XXXX-quality-check-workflow.md

```diff
@@ -0,0 +1,546 @@
+| Per Sequence Quality Scores  | [FastQC] | The number of reads with the indicated average quality score.           | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+| Per Base Sequence Content    | [FastQC] | The bias towards a particular nucleotide at a given position of a read. | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+| Per Sequence GC Content      | [FastQC] | The GC content distribution.                                            | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+| Per Base N Content           | [FastQC] | The percentile of base calls at each position which contained an N.     | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+| Sequence Length Distribution | [FastQC] | The distribution of read lengths in the file.                           | ![no](https://img.shields.io/badge/no-red)           | ![no](https://img.shields.io/badge/no-red)           | ![no](https://img.shields.io/badge/no-red)           |
+| Sequence Duplication Levels  | [FastQC] | The relative level of duplication for each sequence.                    | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+| Overrepresented Sequences    | [FastQC] | The sequences which may be overrepresented in your file.                | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+| Adapter Content              | [FastQC] | The presence or absence of adapter contamination in your file.          | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+| Status Checks                | [FastQC] | An overview of the pass/warn/fail status for all FastQC checks.         | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) | ![yes](https://img.shields.io/badge/yes-brightgreen) |
+
+[FastQC] rounds out the list of important tools to interrogate the quality of
+the sequencing data itself. We leverage many of the plots produced by FastQC,
+using it as the authority on most sequencing quality metrics, including:
+
+- The **sequencing counts** chart, which we typically ignore in favor of
+  [picard]'s mark duplicates plot.
```

Comment:
Another reason that we ignore FastQC's duplicate estimations is that they're produced by the first X reads in the BAM and extrapolated from there. I've seen a decent number of cases where Picard and FastQC produced _wildly_ different estimates for duplication. That disagreement can be explained by sampling bias in FastQC.

I haven't gotten around to it, but I doubt there will be any disagreements as I plan to axe this figure from the reports. It's just inferior in every way to Picard, and should be removed to avoid confusion. IMO it can be probably even be removed the RFC, unless we want to explain why FastQC is unreliable for duplication metrics.